docker-compose.yml

```diff
@@ -10,27 +10,29 @@ services:
   #  command: nc -l 50070
 
   frontier:
-    image: squirrel.frontier:latest
+    image: dicegroup/squirrel.frontier:latest
     container_name: frontier
     environment:
       - HOBBIT_RABBIT_HOST=rabbit
       - URI_WHITELIST_FILE=/var/squirrel/whitelist.txt
       - FRONTIER_CONTEXT_CONFIG_FILE=/var/squirrel/spring-config/frontier-context.xml
-      - SEED_FILE=/var/squirrel/seeds.txt
+      - SEED_FILE=/var/squirrel/seeds.csv
       - MDB_HOST_NAME=mongodb
       - MDB_PORT=27017
-      - QUEUE_FILTER_PERSIST=true
       - MDB_CONNECTION_TIME_OUT=5000
       - MDB_SOCKET_TIME_OUT=10000
       - MDB_SERVER_TIME_OUT=10000
+      - QUEUE_FILTER_PERSIST=true
       - COMMUNICATION_WITH_WEBSERVICE=false
       - VISUALIZATION_OF_CRAWLED_GRAPH=false
-      - JVM_ARGS=-Xmx8g
+      - JVM_ARGS=-Xmx8g 
+      
     volumes:
       - ./data/frontier:/var/squirrel/data
+      - ./seed/seeds.csv:/var/squirrel/seeds.csv:ro
       - ./seed/seeds.txt:/var/squirrel/seeds.txt:ro
       - ./whitelist/whitelist.txt:/var/squirrel/whitelist.txt:ro
-    command: java -cp squirrel.jar org.hobbit.core.run.ComponentStarter org.dice_research.squirrel.components.FrontierComponentStarter
+    command: java -cp squirrel.jar org.dice_research.squirrel.components.FrontierComponentStarter
```

Comment:
Do we really need the commands here? I would suggest to remove them. Since we have different images for Frontier and Worker, they shouldn't be necessary, anymore.

Reply:
yes, you're right, they are no longer needed. I missed that.